### PR TITLE
Creating StampFeesApiAction and StampFeesApiActionImpl

### PR DESCRIPTION
**Pull request by Tauqueer031**

### CODE REVIEW

File: src/main/java/org/ideoholic/curium/model/stampfees/service/StampFeesService.java [hunk 1]
```diff
@@ -301,12 +284,13 @@ private int getLedgerAccountId(String itemAccount) {
 				}
 		    
         		String ItemLedgerId = properties.getProperty(itemAccount);
-		    
+		    System.out.println(itemAccount);
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/stampfees/service/StampFeesService.java]: Remove the Sysout and replace with log.debug

File: src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesApiActionImpl.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+	@GetMapping("/showOtherFeesDetails")
+	public ResponseEntity<OtherFeesDetailsDto> showOtherFeesDetails(@RequestHeader(value = "branchid") String branchId,
+			@RequestHeader(value = "currentAcademicYear") String currentAcademicYear) {
+		OtherFeesDetailsDto otherFeesDetailsDto = new OtherFeesDetailsDto();
+		OtherFeesCategoryResponseDto otherFeesCategoryResponseDto = feesService.viewOtherFees(branchId,
+				currentAcademicYear);
+		Currentacademicyear currentacademicyear = yearService.getYear();
+		ResultResponse result = standardService.viewClasses(branchId);
+		otherFeesDetailsDto.copyOtherFeesCategoryResponseDto(otherFeesCategoryResponseDto);
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesApiActionImpl.java]: `otherFeesDetailsDto.setOtherFeecategory(otherFeesCategoryResponseDto.getOtherFeesCategory());`

File: src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesApiActionImpl.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+		otherFeesDetailsDto.copyCurrentacademicyear(currentacademicyear);
+		otherFeesDetailsDto.copyResultResponse(result);
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesApiActionImpl.java]: `otherFeesDetailsDto.setSuccess(result.isSuccess());`

File: src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesActionAdapter.java [hunk 4]
```diff
@@ -96,4 +93,13 @@ public void advanceSearchForStampFees() {
 		httpSession.setAttribute("feescategory", feescategoryResponseDto.getFeescategory());
 		request.setAttribute("searchStudentList", feescategoryResponseDto.getSearchStudentList());
 	}
+
+	public void multiClassSearch() {
+		SearchStudentDto searchStudentDto = new SearchStudentDto();
+		searchStudentDto.setClassSearch(request.getParameter("classsearch"));
+		searchStudentDto.setNameSearch(request.getParameter("namesearch"));
+		searchStudentDto.setSecSearch(request.getParameter("secsearch"));
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesActionAdapter.java:97]: Here need to set `request.getParameter("academicyear")` also

**Review by reehanlalkhan — CHANGES_REQUESTED** [review]

File: src/main/java/org/ideoholic/curium/model/stampfees/service/StampFeesService.java [hunk 5]
```diff
@@ -301,12 +284,13 @@ private int getLedgerAccountId(String itemAccount) {
...
+        		//log.debug(ItemLedgerId);
```
Comment (reehanlalkhan) [inline src/main/java/org/ideoholic/curium/model/stampfees/service/StampFeesService.java]: `log.debug("The value of ItemLedgerId:{}", ItemLedgerId);`

**Review by reehanlalkhan — CHANGES_REQUESTED** [review]

File: src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesActionAdapter.java (hunk 4 above)
Comment (Tauqueer031) [reply src/main/java/org/ideoholic/curium/model/stampfees/action/StampFeesActionAdapter.java:97]: done

**Review by reehanlalkhan — APPROVED** [review]

**Pull request merged**